### ISSUE TITLE
Setting last author as corresponding author?

### ISSUE THREAD

**Issue by adamrp** [scipy-conference/scipy_proceedings#144]
Currently, whatever author is listed first is the corresponding author. However, we would like to use the *last* author specified as the corresponding author. Will that be possible?

**Comment by katyhuff** [comment]
Let me look into it.

**Comment by katyhuff** [comment]
Ok, I think I can fix this, but not today. If you're willing, please go ahead and make your PR and I'll try to address this issue before the proceedings are published.

**Comment by adamrp** [comment]
Great! I will make the submission now, and fix this later if possible. Thanks again @katyhuff!

**Comment by jackiekazil** [comment]
I was going to ask something similar. Is it possible to get both authors to be corresponding authors? (My colleague is awesome, but I don't want either of us to be a single point of failure.)

**Comment by katyhuff** [comment]
@jackiekazil good idea. I'm sure it's possible, but I'm putting out too many fires this week, so if you wait on me it'll have to wait until shortly before publication. However... if any intrepid pythonista wants to look into it..  here's the offending line of python:  https://github.com/scipy-conference/scipy_proceedings/blob/master/publisher/writer/__init__.py#L127

**Comment by jackiekazil** [comment]
No worries. If I wasn't doing polishing / updates of Mesa code for Scipy I
would totally dig into the code.

**Comment by katyhuff** [comment]
@jackiekazil asked in #133 :+1: 

> Is it possible to include two email addresses or set a group email address as oppose to having this go to one person?

@jackiekazil :  Which of these functionalities do you want? I'm only going to have time to do one of them before the proceedings happen.

**Comment by jackiekazil** [comment]
I would two emails might be the easier of the two. However, we did a quick hack in the paper, where we listed both emails under the first author. So, if that works for you all, I would say that I am satisfied.

**Comment by katyhuff** [comment]
ahh, I see what you're saying. Great!! I'll still try to resolve this issue
at some point soon to help with @adamrp and the Qiita team.

On Fri, Jul 3, 2015 at 4:21 PM, Jackie Kazil <notifications@github.com>
wrote:

> I would two emails might be the easier of the two. However, we did a quick
> hack in the paper, where we listed both emails under the first author. So,
> if that works for you all, I would say that I am satisfied.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/issues/144#issuecomment-118437109>
> .
>



-- 
http://katyhuff.github.com

**Comment by jackiekazil** [comment]
Sounds good! But if it doesn't happen, not the end of the world. I know
these last days can been pretty hectic.

**Comment by adamrp** [comment]
Agreed, we also used a workaround that we are happy with, so maybe we can
just close this issue?

**Comment by jackiekazil** [comment]
:+1: .... unless @katyhuff is really excited making the adjustments OR upset at our hacks. ;-)

**Comment by katyhuff** [comment]
:D Excellent. No hack has ever upset me! I do think this is a worthy
additional feature, so let's leave the issue open, even if it doesn't get
solved until next year. Nice to know that it's all going to be ok even if I
don't get around to it in the next few days, though! thanks.

On Sat, Jul 4, 2015 at 10:37 AM, Jackie Kazil <notifications@github.com>
wrote:

> [image: :+1:] .... unless @katyhuff <https://github.com/katyhuff> is
> really excited making the adjustments OR upset at our hacks. ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/issues/144#issuecomment-118536646>
> .
>



-- 
http://katyhuff.github.com

**Comment by philippjfr** [comment]
We'd like to do something similar? Could one of you let us know what hacks you applied to get two authors to become corresponding authors?

**Comment by jackiekazil** [comment]
Philipp -- What we did was put the email address that we wanted in the
field of the first author.
Can you preview what it looks like here:
http://nipy.bic.berkeley.edu:5000/download/23

**Comment by philippjfr** [comment]
Thanks jackie! @katyhuff Any chance to get proper support for this so that both authors are actually marked as corresponding authors in the title?

**Comment by katyhuff** [comment]
I'm happy to try to make it so that the corresponding author can be at any place in the author list... however, traditionally, the corresponding author exists specifically in order to delineate a single point of contact for the paper (rather than all the authors). So, it seems a bit odd to me to have two or more. Maybe I'm missing something about the motivation?

**Comment by philippjfr** [comment]
Really it's more about indicating that we are both joint first authors. We'd very much welcome an alternative way of indicating this joint status.

**Comment by stefanv** [comment]
The corresponding author is not elevated in any way.  How do other journals indicate joint status?  I don't think I've seen that done before.

**Comment by philippjfr** [comment]
I've had trouble with this myself but it seems the journal either explicitly states somewhere that both authors made equal contributions to the paper or both first authors are marked as corresponding authors.

**Comment by stefanv** [comment]
I can implement either for you--let me know what you prefer.

**Comment by katyhuff** [comment]
@stefanv : I have a plan that should work to achieve both goals, if you
want to give me the afternoon to get around to it, I'd been planning to.



On Mon, Jul 6, 2015 at 10:52 AM, Stefan van der Walt <
notifications@github.com> wrote:

> I can implement either for you--let me know what you prefer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/issues/144#issuecomment-118906284>
> .
>



-- 
http://katyhuff.github.com

**Comment by philippjfr** [comment]
I'd be happy with either but explicitly stating that both authors contributed equally would probably be best. Thanks a lot for working on this!

**Comment by stefanv** [comment]
Certainly! Just shout if you need my help with anything.

**Comment by katyhuff** [comment]
Thanks @stefanv ! We've merged in a solution to this with #171. It's now been merged into the 2015 branch as well, so you can use `:corresponding:` to make the corresponding author someone other than the first author : 

```
:author: Ulysses the Uncommunicative
:email: jj@rome.it
:institution: Senate House, S.P.Q.R.

:author: Terry Talkative
:email: mark37@rome.it
:institution: Egyptian Embassy, S.P.Q.R.
:corresponding:
```
or use `:equal-contributor:` to make a footnote that some subset of authors contributed equally:

```
:author: Gaius Caesar
:email: jj@rome.it
:institution: Senate House, S.P.Q.R.
:equal-contributor:

:author: Mark Anthony
:email: mark37@rome.it
:institution: Egyptian Embassy, S.P.Q.R.

:author: Brutus
:email: brutus@rome.it
:institution: Unaffiliated
:equal-contributor:
```

**Comment by philippjfr** [comment]
Just tried applying this to our paper and it seems to duplicate this text:

```
† These authors contributed equally.
```

See [here](http://nipy.bic.berkeley.edu:5000/download/27)

The example above seems to indicate that you should add the tag to all equal contributors. Any easy way to fix this?

**Comment by katyhuff** [comment]
@philippjfr Thanks for pointing that out. [Just fixed it](http://nipy.bic.berkeley.edu:5000/download/27).

**Comment by philippjfr** [comment]
Thanks for fixing it! Probably need to trigger a new build of our paper somehow for the change to propagate.

**Comment by katyhuff** [comment]
Already done when I sent the message. You may have the doc cached in your
browser though, so hit refresh to avoid that browerside caching.

On Wed, Jul 8, 2015 at 12:32 AM, Philipp Rudiger <notifications@github.com>
wrote:

> Thanks for fixing it! Probably need to trigger a new build of our paper
> somehow for the change to propagate.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scipy-conference/scipy_proceedings/issues/144#issuecomment-119442042>
> .
>



-- 
http://katyhuff.github.com
